readthedocs/settings/celery_nodb.py

```diff
@@ -0,0 +1,25 @@
+"""Local setting for celery without DB access"""
+
+import os
+
+from .dev import CommunityDevSettings
```

Comment:
If this config is branched off of dev, i think a more appropriate module name would be `readthedocs.settings.dev.celery`, `celery_nodb` is disconnected from the dev vs base vs production distinction.

Reply:
I moved the `readthedocs/settings/dev.py` to `readthedocs/settings/dev/__init__.py` so the python path to the file does not change.

Besides, I added `readthedocs.settings.dev.celery.*` namespace with `builds` and `webs` on it, so it's easy to know which one is the correct one on each celery process.

Reply:
This was in conflict with readthedocsinc naming scheme, which would be `readthedocs.settings.build.dev`. I changed it to keep things consistent and more easily understandable between the two repos.